src/BudgetManager/BudgetManager.CQRS/Responses/NotificationResponses/NotificationRespone.cs

```diff
@@ -1,11 +1,15 @@
 ﻿using BudgetManager.Model.Enums;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace BudgetManager.CQRS.Responses.NotificationResponses
 {
-    public record NotificationResponse(Guid Id, NotificationTypes NotificationType, string Description, bool IsRead);
```

Comment:
This was not the problem. The problem is that come notification handlers return wrong notification, like UpdateNotification returns FirstOrDefault.